corehq/apps/analytics/templates/analytics/initial/drift.html

```diff
@@ -1,7 +1,9 @@
 {% load hq_shared_tags %}
-{% if is_saas_environment and not request|toggle_enabled:"USER_TESTING_SIMPLIFY" %}
-  {% if not request.user.is_authenticated or request.couch_user.days_since_created < 31 %}
-    {# This was already publicly visible, best practice: todo get into settings #}
-    {% initial_analytics_data 'drift.apiId' '7456r8nz2u34' %}
+{% if not request|toggle_enabled:"USER_TESTING_SIMPLIFY" %}
+  {% if is_saas_environment or ANALYTICS_CONFIG.DEBUG %}
```

Comment:
Why not include this in the if statement above? Also `is_saas_environment` and the debug check should come probably come before toggle_enabled since toggles are stored in the DB and these other two are in localsettings (so in local memory)

Reply:
Could it be rewritten as:

```
if analytics.debug:
    include 
elif not is_saas_environment:
   don't include
elif USER_TESTING_SIMPLIFY:
  don't include
elif user not authed or user created < 31 days:
   include
```

the 3 nests of if statements is getting a bit confusing for me and the django template syntax doesn't help.

Reply:
https://github.com/dimagi/commcare-hq/pull/18841/commits/b003ac9f61dd386da6c14fa09fd5e81b8bfb5c3a better?